packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -242,7 +242,7 @@ export const getMaxMin = (
             isSddFlow ? method.limits.min : limitMinAmount
           ).toString()
 
-          const minFiat = !limitMinChanged ? convertBaseToStandard('FIAT', min) : min
+          const minFiat = convertBaseToStandard('FIAT', min)
```

Comment:
I think this fix was previously introduced to prevent some edge case, we just need to test all possible edge cases.

Reply:
Do you remember which one @milan-bc ?

Reply:
The only time `limitMinChanged` is set to true is when `limits?.minOrder` exists and is greater than `pair.buyMin`. From what I could see limits.minOrder is coming back from the api in the standard format (life would be easier if it was just base format). Where this might break is if the api returns base format sometimes and not other times or if `isSddFlow` is true and that causes something weird to happen.

Reply:
@pedroapfilho nah, they sometimes pops up like that after someone reported it